lib/tasks/factorybot.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+desc "Run Factroybot linter"
+task factorybot_lint: :environment do
```

Comment:
I think we could make this more conventional to rake if this was a task of lint under the namespace of factorybot. So task becomes `factorybot:lint`.

If you check our tasks with `rake -T` to compare to others that might help